<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="3" passed="1" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-08-07T17:16:43Z" name="MyExtentDemoSuite" finished-at="2022-08-07T17:17:07Z" duration-ms="23897">
    <groups>
    </groups>
    <test started-at="2022-08-07T17:16:43Z" name="ExtentReport_Test" finished-at="2022-08-07T17:17:07Z" duration-ms="23897">
      <class name="day39.extentreports.LoginTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:day39.extentreports.LoginTest@27ff5d15]" started-at="2022-08-07T22:46:43Z" name="setup" finished-at="2022-08-07T22:46:57Z" duration-ms="13414" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="testLogo()[pri:0, instance:day39.extentreports.LoginTest@27ff5d15]" started-at="2022-08-07T22:46:57Z" name="testLogo" finished-at="2022-08-07T22:46:57Z" duration-ms="173" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogo -->
        <test-method signature="testLogin()[pri:0, instance:day39.extentreports.LoginTest@27ff5d15]" started-at="2022-08-07T22:46:57Z" name="testLogin" finished-at="2022-08-07T22:47:06Z" duration-ms="9407" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:489)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:281)
	at org.testng.Assert.assertEquals(Assert.java:291)
	at day39.extentreports.LoginTest.testLogin(LoginTest.java:58)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:715)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:907)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1237)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:51)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:85)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogin -->
        <test-method signature="testLogout()[pri:0, instance:day39.extentreports.LoginTest@27ff5d15]" started-at="2022-08-07T22:47:06Z" name="testLogout" finished-at="2022-08-07T22:47:06Z" depends-on-methods="day39.extentreports.LoginTest.testLogin" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogout -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:day39.extentreports.LoginTest@27ff5d15]" started-at="2022-08-07T22:47:06Z" name="tearDown" finished-at="2022-08-07T22:47:07Z" duration-ms="444" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- day39.extentreports.LoginTest -->
    </test> <!-- ExtentReport_Test -->
  </suite> <!-- MyExtentDemoSuite -->
</testng-results>
